### PR TITLE
Work around npm bug that install @mdx-js/react twice resulting in empty Context and unstyled tables

### PR DESCRIPTION
**Pull request by nsams**
https://github.com/npm/cli/issues/3477

Moving those from devDependencies to dependencies doesn't look that wrong, if we consider local dev as dev (where
@types are needed) and the netlify deployment of storybook as prod.

In pratice there is no difference, because we (apperently) use dev dependencies everywhere

### CODE REVIEW

File: packages/admin-stories/package.json [hunk 1]
```diff
@@ -29,12 +34,6 @@
     },
     "devDependencies": {
         "@babel/core": "^7.8.3",
-        "@storybook/addon-actions": "^6.2.7",
-        "@storybook/addon-docs": "^6.2.7",
```
Comment (johnnyomair) [inline packages/admin-stories/package.json:33]: Why did you remove this dependency?

**Comment by nsams** [comment]
ok that was stupid, sorry

**Comment by nsams** [comment]
now it makes more sense :)

**Review by dkarnutsch — APPROVED** [review]

**Review by rabengraph — APPROVED** [review]
